openfl/experimental/component/aggregator/aggregator.py

```diff
@@ -363,22 +366,21 @@ def do_task(self, f_name: str) -> Any:
 
             # Transition check
             if aggregator_to_collaborator(f, parent_func):
```

Comment:
Should there be any special treatment of private attributes in the opposite case (`collaborator_to_aggregator`)?

Reply:
I'm asking, because in the experimental `collaborator.py`, for some reason we call `__delete_agg_attrs_from_clone` instead of `__delete_collab_attrs_from_clone`:
https://github.com/securefederatedai/openfl/blob/develop/openfl/experimental/component/collaborator/collaborator.py#L202

Reply:
The private attribute in opposite case (collaborator_to_aggregator) are managed inside collaborator so there is no need for special handling. There seems to be an incorrect naming in https://github.com/securefederatedai/openfl/blob/develop/openfl/experimental/component/collaborator/collaborator.py#L202, 
`__delete_agg_attrs_from_clone` should be actually named to `__delete_collab_attrs_from_clone`
 
Thanks for pointing this out, Our proposal is to rename the functions to and do some code harmonization

`__delete_agg_attrs_from_clone`: Should be renamed to `__delete_private_attrs_from_clone`
`__set_attributes_to_clone`: Should be renamed to `__set_private_attrs_to_clone`